clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -1405,9 +1394,8 @@ protected static boolean shouldBuildDictionary(PushJobSetting pushJobSetting, St
     }
 
     LOGGER.info(
-        "No Compression dictionary will be generated with the strategy {} and compressionMetricCollectionEnabled is {}",
-        storeSetting.compressionStrategy,
-        (pushJobSetting.compressionMetricCollectionEnabled ? "Enabled" : "Disabled"));
```

Comment:
Because the value of `compressionMetricCollectionEnabled` is always false, when execute this line.

See the original codes
```
    if (pushJobSetting.compressionMetricCollectionEnabled
        || storeSetting.compressionStrategy == CompressionStrategy.ZSTD_WITH_DICT) {
      if (pushJobSetting.isIncrementalPush) {
        // do something
      }
     // at this moment, compressionMetricCollectionEnabled is always false
      LOGGER.info(
          "Compression dictionary will be generated with the strategy {} and compressionMetricCollectionEnabled is {}",
          storeSetting.compressionStrategy,
          (pushJobSetting.compressionMetricCollectionEnabled ? "Enabled" : "Disabled"));
      return true;
    }
```